students/hyska_monika/lesson_09_reading_and_writing_files/mcb.pyw

```diff
@@ -0,0 +1,36 @@
+"""
+Multiclipboard program
+commands(sample commands are on mcb.bat file):
+py [filename].pyw save [keyword]
+    -- save copied text to clipboard with special keyword
+py [filename].pyw [keyword]
+    -- paste to clipboard  text for saved keyword
+py [filename].pyw list
+    -- copy keywords list
+py [filename].pyw delete [keyword]
+    -- delete saved text with keyword
+py [filename].pyw delete all
+    -- clear all keyword
+"""
+import shelve, pyperclip, sys
+
+
+mcbShelf = shelve.open("mcb")
+keyword = sys.argv
+command = keyword[1].lower()
+sub_command = keyword[2].lower()
```

Comment:
Better use unpacking `__, command, subcommand = sys.argv` (`__` stands for ignored). Besides if you try to access `keyword[2]` when `len(keyword) == 2` it should crash with `IndexError`. Have you tested that?